src/librustc_typeck/check/method/suggest.rs

```diff
@@ -324,6 +324,12 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
                                 );
                             }
                         }
+                        if let ty::RawPtr(_) = &actual.sty {
+                            err.note("you might want to use `<*const T>::as_ref()` to get a \
```

Comment:
```suggestion
                            err.note("try using `<*const T>::as_ref()` to get a \
```
A bit more direct & succinct =)

Reply:
done